tests/unit/misc/test_guiprocess.py

```diff
@@ -60,7 +60,7 @@ def test_start(proc, qtbot, message_mock, py_proc):
         proc.start(*argv)
 
     assert not message_mock.messages
-    assert bytes(proc._proc.readAll()).rstrip() == b'test'
+    assert qutescheme.spawn_output == proc.spawn_format(0, 0, stdout="test")
```

Comment:
No need to pass `0, 0` explicitly here, as it's the default anyways, and `stdout` is a named argument.